### PR TITLE
test(android_intent_plus): add e2e test

### PR DESCRIPTION
**Pull request by Chever-John**
## Description

I am adding e2e test cases to the examples of the android_intent_plus plugin.

## Related Issues

It should be related to [this issues](https://github.com/fluttercommunity/plus_plugins/issues/958)

#958

## Checklist

Before you create this PR confirm that it meets all requirements listed below by checking the relevant checkboxes (`[x]`).
This will ensure a smooth and quick review process.

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] My PR includes unit or integration tests for *all* changed/updated/fixed behaviors (See [Contributor Guide]).
- [x] All existing and new tests are passing.
- [x] I updated the version in `pubspec.yaml` and `CHANGELOG.md`.
- [ ] I updated/added relevant documentation (doc comments with `///`).
- [x] The analyzer (`flutter analyze`) does not report any problems on my PR.
- [x] I read and followed the [Flutter Style Guide].
- [x] I am willing to follow-up on review comments in a timely manner.

## Breaking Change

Does your PR require plugin users to manually update their apps to accommodate your change?

- [ ] Yes, this is a breaking change (please indicate a breaking change in CHANGELOG.md and increment major revision).
- [x] No, this is *not* a breaking change.

[issue database]: https://github.com/flutter/flutter/issues
[Contributor Guide]: https://github.com/fluttercommunity/plus_plugins/blob/main/CONTRIBUTING.md
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[pub versioning philosophy]: https://dart.dev/tools/pub/versioning

### CODE REVIEW

**Comment by miquelbeltran** [comment]
Thanks! 

One thing, you should rename `android_intent_plus_e2e.dart` to `android_intent_plus_test.dart`.

Then, please confirm that the tests work, by running `flutter test integration_test/android_intent_plus_test,dart`

As you can see, the step `all_plugins / integration_tests (pull_request)` is Skipped. This means that GitHub Actions is not running integration tests at the moment. We will work on enabling those afterwards.

File: packages/android_intent_plus/example/integration_test/android_intent_plus_e2e.dart [hunk 1]
```diff
@@ -1,18 +1,19 @@
 import 'dart:io';
```
Comment (miquelbeltran) [inline packages/android_intent_plus/example/integration_test/android_intent_plus_e2e.dart:1]: Please rename the file to `android_intent_plus_test.dart`

**Review by miquelbeltran — CHANGES_REQUESTED** [review]

**Comment by miquelbeltran** [comment]
@Chever-John I have pushed a change on your code branch, I have enabled the integration tests to run on GitHub Actions, but only for Android at the moment. I want to see if this works.

Can you fix the naming, as I explained in the previous comment?

**Comment by Chever-John** [comment]
@miquelbeltran 
Oh, that is what I thought. Enable the integration tests to run on GitHub Actions~

File: packages/android_intent_plus/example/integration_test/android_intent_plus_e2e.dart (hunk 1 above)
Comment (Chever-John) [reply packages/android_intent_plus/example/integration_test/android_intent_plus_e2e.dart]: I think I have finished it :D

**Review by miquelbeltran — APPROVED** [review]

**Comment by Chever-John** [comment]
@miquelbeltran 
I've been following this PR.
Indeed, it's a great work :tada:

Thanks ~

**Pull request merged**